coral-component-table/src/scripts/Table.js

```diff
@@ -1616,6 +1622,20 @@ class Table extends BaseComponent(HTMLTableElement) {
       // For icons (chevron up/down) styling
       headerCell.setAttribute('sortabledirection', column.sortableDirection);
       table._getActiveHeader(headerCell).setAttribute('aria-sort', column.sortableDirection === sortableDirection.DEFAULT ? 'none' : column.sortableDirection);
+      const liveRegion = this._elements.liveRegion;
+      if (liveRegion) {
+        if (column.sortableDirection === sortableDirection.DEFAULT) {
+          liveRegion.innerText = '';
+        }
+        else {
+          if (headerCell.content.textContent.trim().length) {
+            liveRegion.innerText = i18n.get('sorted by column {0} in ' + column.sortableDirection + ' order', headerCell.content.textContent);
+            window.requestAnimationFrame(() => {
+              liveRegion.setAttribute('aria-live', 'off');
+            });
```

Comment:
I think this needs to be a `setTimeout`with some delay to give assistive technology time to pick up on the content change. Otherwise, I don't think the liveRegion will announce. Setting `aria-live="off"` tells AT to ignore updates to the live region.

Reply:
@majornista ah too bad, this was merged already .. can you please submit a PR to change it to `setTimeout` ideally with a comment in the code to explain why it's needed.